docs/products/kafka/howto/avoid-high-resource-usage-on-kafka-startup-plan.rst

```diff
@@ -0,0 +1,48 @@
+Avoid high resource usage on Kafka® ``Startup-2`` plan
+======================================================
+
+The Kafka Startup-2 Plan has been optimized for lightweight operations, making it ideal for applications that handle fewer messages per second and don't demand high throughput. But sometimes, you might encounter an alert showing high resource usage. This alert is generally triggered when the Kafka broker memory drops too low and CPU idle time is less than 15%. Grasping why these alerts occur and how to handle them will aid in optimizing your Kafka usage and maintaining application performance.
+
+What triggers high resource usage?
+----------------------------------
+
```

Comment:
Add before bullet list: 
There are a few things that can trigger high resource usage on the Kafka Startup-2 plan: